bridge/contracts/Snapshots.sol

```diff
@@ -160,38 +166,58 @@ contract Snapshots is Initializable, SnapshotsStorage, ISnapshots {
         return true;
     }
 
+    /// @notice Gets snapshot desperation factor
+    /// @return The snapshot desperation factor
     function getSnapshotDesperationFactor() public view returns (uint256) {
         return _snapshotDesperationFactor;
     }
 
+    /// @notice Gets snapshot desperation delay
+    /// @return The snapshot desperation delay
     function getSnapshotDesperationDelay() public view returns (uint256) {
         return _snapshotDesperationDelay;
     }
 
+    /// @notice Gets minimal interval between snapshots
+    /// @return The minimal interval between snapshots
     function getMinimumIntervalBetweenSnapshots() public view returns (uint256) {
         return _minimumIntervalBetweenSnapshots;
     }
 
+    /// @notice Gets the chain Id
+    /// @return The chain Id
     function getChainId() public view returns (uint256) {
         return _chainId;
     }
 
+    /// @notice Gets the epoch from epoch register
+    /// @return The epoch from epoch register
     function getEpoch() public view returns (uint256) {
         return _epochRegister().get();
     }
 
+    /// @notice Gets the epoch length
+    /// @return The epoch length
     function getEpochLength() public view returns (uint256) {
         return _epochLength;
     }
 
+    /// @notice Gets the chain Id from snapshot for specified epoch
```

Comment:
Can we add that the function will fail in case the user tries to get a snapshot older than 6 epochs from the current one?

Reply:
Done